services/web/server/src/simcore_service_webserver/long_running_tasks.py

```diff
@@ -17,8 +17,8 @@
 
 
 class _RequestContext(BaseModel):
-    user_id: UserID = Field(..., alias=RQT_USERID_KEY)
-    product_name: str = Field(..., alias=RQ_PRODUCT_KEY)
+    user_id: UserID = Field(..., alias=RQT_USERID_KEY)  # type:ignore
+    product_name: str = Field(..., alias=RQ_PRODUCT_KEY)  # type:ignore
```

Comment:
Q: why are these ignored?

Reply:
```cmd
src/simcore_service_webserver/long_running_tasks.py:20:5: error: Required dynamic aliases disallowed  [pydantic-alias]
src/simcore_service_webserver/long_running_tasks.py:21:5: error: Required dynamic aliases disallowed  [pydantic-alias]
```

But in reality they are not dynamic, they are `Final` constants